src/main/kotlin/org/pkl/lsp/PklLSPServer.kt

```diff
@@ -133,15 +120,6 @@ class PklLSPServer(val verbose: Boolean) : LanguageServer {
   @Suppress("unused")
   @JsonRequest(value = "pkl/syncProjects")
   fun syncProjects(@Suppress("UNUSED_PARAMETER") ignored: Any?): CompletableFuture<Unit> {
-    return project.pklProjectManager.syncProjects()
-  }
-
-  private fun cacheStdlib() {
-    stdlibDir.toFile().mkdirs()
-    for ((name, _) in project.stdlib.allModules()) {
-      val file = stdlibDir.resolve("$name.pkl")
-      val text = IoUtils.readClassPathResourceAsString(javaClass, "/org/pkl/core/stdlib/$name.pkl")
-      Files.writeString(file, text, Charsets.UTF_8)
-    }
```

Comment:
Since the `VirtualFile`s themselves are now responsible for building, we don't really need to cache the stdlib anymore. Instead, we can just build them on demand, like we do any other module.